pyvista/core/utilities/reader.py

```diff
@@ -2831,6 +2839,259 @@ class ProStarReader(BaseReader):
+
+        Parameters
+        ----------
+        name : str
+            Point array name.
+
+        """
+        self.reader.SetPointResultArrayStatus(name, 0)
+
+    def point_array_status(self, name):
+        """Get status of point array with name.
+
+        Parameters
+        ----------
+        name : str
+            Point array name.
+
+        Returns
+        -------
+        bool
+            Whether reading the cell array is enabled.
+
+        """
+        return bool(self.reader.GetPointResultArrayStatus(name))
+
+    @property
+    def number_cell_arrays(self):
+        """Return the number of point arrays.
+
+        Returns
+        -------
+        int
+            Number of point arrays.
+
+        """
+        return self.reader.GetNumberOfElementResultArrays()
+
+    @property
+    def cell_array_names(self):
+        """Return the list of all cell array names.
+
+        Returns
+        -------
+        list[str]
+            List of all cell array names.
+
+        """
+        return [self.reader.GetElementResultArrayName(i) for i in range(self.number_cell_arrays)]
+
+    def enable_cell_array(self, name):
+        """Enable cell array with name.
+
+        Parameters
+        ----------
+        name : str
+            Cell array name.
+
+        """
+        self.reader.SetElementResultArrayStatus(name, 1)
+
+    def disable_cell_array(self, name):
+        """Disable cell array with name.
+
+        Parameters
+        ----------
+        name : str
+            Cell array name.
+
+        """
+        self.reader.SetElementResultArrayStatus(name, 0)
+
+    def cell_array_status(self, name):
+        """Get status of cell array with name.
+
+        Parameters
+        ----------
+        name : str
+            Cell array name.
+
+        Returns
+        -------
+        bool
+            Whether reading the cell array is enabled.
+
+        """
+        return bool(self.reader.GetElementResultArrayStatus(name))
+
+    @property
+    def time_values(self):
+        """All time or iteration values.
+
+        Returns
+        -------
+        list[float]
+
+        """
+        vtkStreaming = _lazy_vtk_instantiation(
+            'vtkCommonExecutionModel', 'vtkStreamingDemandDrivenPipeline'
+        )
+        key = vtkStreaming.TIME_STEPS()
+        vtkinfo = self.reader.GetExecutive().GetOutputInformation(0)
+        return [vtkinfo.Get(key, i) for i in range(self.number_time_points)]
+
+    def time_point_value(self, time_point):
+        """Value of time point or iteration by index.
+
+        Parameters
+        ----------
+        time_point : int
+            Time point index.
+
+        Returns
+        -------
+        float
+
+        """
+        return self.time_values[time_point]
+
+    @property
+    def active_time_value(self):
+        """Active time or iteration value.
+
+        Returns
+        -------
+        float
+
+        """
+        return self.time_values[self._active_time_point]
```

Comment:
Can we use `GetTimeStep` here instead of having a private attribute?  It is always better to use the vtk method when possible because someone could directly modify the vtk object, and it wouldn't be updated on the PyVista class private attribute.